components/brave_rewards/browser/rewards_service.h

```diff
@@ -240,6 +243,10 @@ class RewardsService : public KeyedService {
       const std::string& type,
       const std::map<std::string, std::string>& args,
       GetShareURLCallback callback) = 0;
+  virtual void GetMonthlyStatements(
+      int32_t month,
```

Comment:
Is there a reason that month is signed and year is unsigned?

Reply:
Yes, month maps to `ledger::MONTH` which can be -1 for `ANY`.

Reply:
OK, gotcha.